test/web-platform-tests/run-wpts.js

```diff
@@ -18,12 +18,15 @@ const validReasons = new Set([
   "needs-node10",
   "needs-node11",
   "needs-node12",
+  "needs-node13",
```

Comment:
Please don't add a new fail reason that's not in use. We should be trying to drive down the number of Node-version-specific failures, not increase them.